tests/Makefile

```diff
@@ -8,6 +8,17 @@ endif
 # Attempt to guess where to find various tools and libraries
 ################################################################
 
+MODE ?= release
+ifeq ($(MODE),release)
+	UC_MODE := Release
+else ifeq ($(MODE),debug)
+	UC_MODE := Debug
+else ifeq ($(MODE),fast)
+	UC_MODE := Fast
+else
+	$(error "Mode must be 'debug' or 'release'")
+endif
```

Comment:
I have no idea why that's there either, but do note that it does permit the mode to be "fast"; it just doesn't say it in the error message.